### PR TITLE
add FR locale

### PR DESCRIPTION
**Pull request by Tukks**
Add fr locale to work with French Alexa.
tested and working.

Just need to set you phone in English to be able to configure Alexa Assistant en your phone.

### CODE REVIEW

**Comment by ettio54** [comment]
Have you managed to make google assistant speak french through alexa ?

**Comment by Tukks** [comment]
Yes, it's working now

**Comment by ettio54** [comment]
I put my phone in english US, set alexa assistant to french. I can speak to the assistant in french but it will respond in english. Did you modified a file to add the language ?

**Comment by Tukks** [comment]
You need to add 'fr-FR',
line 328` if (locale == 'en-GB' || locale == 'de-DE' || locale == 'en-AU' || locale == 'en-CA' || locale == 'en-IN' || locale == 'ja-JP' || locale == 'fr-FR'){`
in the skill code.
Download the github repo, modify index.js and change locale, zip the project, go to Amazon Lambda et reupload the project.

**Comment by ettio54** [comment]
To reupload the project, is it in uploading a file in function code ?

**Comment by Tukks** [comment]
zip all the project again, and upload the zip file.

**Comment by ettio54** [comment]
I zipped the project with the modified index but I can't find where to upload it in lambda. I used the cloudformation to setup. 
Sorry for the amount of questions I'm asking, it's the first time I use aws.

**Comment by Tukks** [comment]
Right now, I don't have access to AWS, I will describe the step tomorrow.

**Comment by ettio54** [comment]
No problem, thanks a lot for the help

**Comment by ettio54** [comment]
Problem solved ! I managed to upload the file. Thanks again

**Comment by MehdyDriouech** [comment]
hello, i would like to know where we are supposed to update the zip file please :)

**Comment by ettio54** [comment]
I went to amazon lambda into functions. Then into the alexa function. In actions, exported the function modified the file and uploaded it back in function code.

**Comment by MehdyDriouech** [comment]
It works ! Thanks for the Info ettio54, and thanks tukks for the fix :)

**Comment by aldialloMC** [comment]
Hi,

I've tried the above indication but I'm getting and error from Alexa stating that there's been an error with the requested skill; any idea of what can be the issue? I modified the file as suggested above and save it before ziping the whole function folder.

**Comment by aldialloMC** [comment]
Hello,

Actually, this is what Alexa is replying when I asked to ask google for something: "There was a Problem with Requested Skills Response"

as I mentioned previously , I modified the Index.js file and uploaded it back so not sure what I'm doing wrong.

Just to be sure that I did the zip file download correctly, once  I click on the export function button a new window pop up giving me the choice to close, download AWS SAM file or Download deployment package. I clicked on Download deployment package. got  a Zip file of 18.9MB, unzipped the file, modified the required lines and saved it before zipping back the file and uploading it from the function code section where it labeled function package upload.  

All the other options of the Function code section were left unchanged, and from the moment I do that and the zip file is uploaded I get the: "There was a Problem with Requested Skills Response" message.

I've tried the common problem page, especially the section: When I run the skill on my Echo Show/Spot I get the error ""There was a Problem with Requested Skills Response" but the Amazon page is completely different from the print screen provided and I cannot find the skill information tab.

So basically I'm currently back to square one, where the I ask a question in french and I get a response in English which is not very helpful for the wife and kids.

So if some one could point out what I'm doing wrong that would be great! 

Thanks in advance

**Comment by BanHop** [comment]
> Add fr locale to work with French Alexa.
> tested and working.
> 
> Just need to set you phone in English to be able to configure Alexa Assistant en your phone.
> Referencing #56
> Edit : added ITalian too

Alexa is in Spanish, i add es-ES locale, but speak google assistant in English

**Comment by Tukks** [comment]
update: I removed the if to allow all locale

**Comment by 1simone0** [comment]
Hi, 
I spent much time to try to upload the modified file but I still don't understand where should Iupload it... I feel so stupid !!!
If I don't wrong I should download and unzip the whole repository on my pc and modify the index file. Then re-zip the whole project and upload somewhere but I can't find the "alexa function" you are saying...
Can you @Tukks @ettio54 @MehdyDriouech help me explaining step by step and posting some urls if you can?
Thank you guys

**Comment by 1simone0** [comment]
anyone?
